docs: correct for extensions API deprecation (#24159)
electron/electron: docs/tutorial/devtools-extension.md

     * `~/.config/google-chrome-canary/Default/Extensions/`
     * `~/.config/chromium/Default/Extensions/`
   * on macOS it is `~/Library/Application Support/Google/Chrome/Default/Extensions`.
1. Pass the location of the extension to `ses.loadExtension`
   API, for the React Developer Tools, it is something like:
   
   ```javascript
   const { app, session } = require('electron')
   const path = require('path')
   app.on('ready', async () => {
     const installedExtension = await session.defaultSession.loadExtension(path.join(__dirname, 'react-devtools'));
     // Note that in order to use the React DevTools extension, you'll need to

>      // download and unzip a copy of the extension.

Reviewer: Seems like this should be a list item between "locate the extension" and "load the extension", rather than a comment.

   })
   ```

The extension will be remembered so you only need to call this API once per
extension. If you try to add an extension that has already been loaded, this method
will not return and instead log a warning to the console.

> 

Reviewer: Looks like an extra newline.

```suggestion
```

### How to remove a DevTools Extension

> You can pass the Id of the extension to the `BrowserWindow.removeExtension`
> API to remove it. 
>    ```javascript
>       ses.removeExtension(extensionId)

Reviewer: How do you find the extension ID? Is it an attribute on the `installedExtension` from above?

```suggestion
Use `ses.removeExtension` with an extension ID to remove the specified extension.

```javascript
 session.defaultSession.removeExtension(extensionId)
```

>    ```

Reviewer 2: I think this section can be removed, as unlike the old API, the extension doesn't stay loaded if you stop calling `loadExtension`.

### See Also 

> You can also refer to other methods for more information about extensions.
> [Loading Extensions using Session](https://www.electronjs.org/docs/api/session#sesloadextensionpath)

Reviewer: ```suggestion
* [API documentation for loading Extensions within a Session](../api/session.md#sesloadextensionpath)
```
